TeamMemberFiles/darien.dalton.txt

```diff
@@ -0,0 +1,5 @@
+Darien Dalton
+Please call me Darien.
+I'm in the section 294.
+I have most of my knowledge in Java and have some knowledge in Python and SQL.
+I am a full time student/parent and I work for DPS.
```

Comment:
Excellent work Darien.
Please clean up your stuff on your side to protect your information